modules/apps/translation/translation-web/src/main/java/com/liferay/translation/web/internal/display/context/ViewTranslationDisplayContext.java

```diff
@@ -122,18 +125,21 @@ public Locale getSourceLocale() {
 		return _translationSnapshot.getSourceLocale();
 	}
 
-	public String getStringValue(InfoField infoField, Locale locale) {
+	public List<String> getStringValues(InfoField infoField, Locale locale) {
 		InfoItemFieldValues infoItemFieldValues =
 			_translationSnapshot.getInfoItemFieldValues();
 
-		InfoFieldValue<Object> infoFieldValue =
-			infoItemFieldValues.getInfoFieldValue(infoField.getName());
+		Collection<InfoFieldValue<Object>> infoFieldValues =
+			infoItemFieldValues.getInfoFieldValues(infoField.getName());
 
-		if (infoFieldValue != null) {
-			return GetterUtil.getString(infoFieldValue.getValue(locale));
-		}
+		Stream<InfoFieldValue<Object>> stream = infoFieldValues.stream();
```

Comment:
now, this should be a for loop.

Reply:
Hi @AliciaGarciaGarcia thanks for your input!
I can do that, although I followed the code pattern at https://github.com/liferay/liferay-portal/blob/master/modules/apps/translation/translation-web/src/main/java/com/liferay/translation/web/internal/display/context/TranslateDisplayContext.java#L359-L364
Should I still implement it as a for loop?

Reply:
Sincerely I do not know because there is a message about using loops instead streams.

Reply:
Hi @AliciaGarciaGarcia I haven't seen SF messages about that. Could you please show me the message?
Thanks!